lua/gitlab/state.lua

```diff
@@ -202,6 +240,135 @@ M.unlinked_discussion_tree = {
   unresolved_expanded = false,
 }
 
+-- These keymaps are set globally when the plugin is initialized
+M.set_global_keymaps = function()
+  local keymaps = M.settings.keymaps
+
+  if keymaps.disable_all or keymaps.global.disable_all then
+    return
+  end
+
+  if keymaps.global.start_review then
+    vim.keymap.set("n", keymaps.global.start_review, function()
+      require("gitlab").review()
+    end, { desc = "Start Gitlab review", nowait = keymaps.global.start_review_nowait })
+  end
+
+  if keymaps.global.choose_merge_request then
```

Comment:
Agreed that this can be dried up but no need to do that now, it's not really too broken, would rather have some repetition here than some fancy abstraction that makes it hard to change these in the future.